index.js

```diff
@@ -52,10 +53,13 @@ class Provider {
   }
 
   _processArgs(argString) {
+    debug (`processing arg: ${argString}`)
```

Comment:
I think it looks better without the space between the function name and the function call. 
`debug('string here')`

Same for L61 and L90.